gatsby-node.js

```diff
@@ -12,6 +13,12 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
       name: `slug`,
       value: slug,
     })
+    const lastUpdatedAt = execSync(`git log -1 --pretty=format:%aI ${node.fileAbsolutePath}`).toString()
```

Comment:
Looks like `git log -1 --pretty=format:%aI` is enough  to get the timestamp of the last commit.